sos/db/src/main/java/se/inera/intyg/common/db/v1/rest/DbModuleApiV1.java

```diff
@@ -110,4 +111,17 @@ public String getAdditionalInfo(Intyg intyg) throws ModuleException {
         return "";
```

Comment:
This was not changed but seems a kittke bit odd...

Reply:
I looked at the commit and it seems to be from 2017. The getAdditionalInfo seems to only contain an implementation for some certificate types. The comment seems to have a copy/paste error as it refers to DOI.